spec/complete_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe Conjur::CLI::Complete do
+describe Conjur::CLI::Complete, wip: true do
```

Comment:
Is anyone even using our completions? I'm not. 

The OSX CLI package activates them by default, maybe? It doesn't seem like it though, because it doesn't complete in my zsh.

`wip` is probably right: we may want completions but it doesn't seem to matter right now? Is there a better built-in tag or something we could use for this idea?